src/PrestaShopBundle/Resources/config/services/bundle/form/form_data_provider.yml

```diff
@@ -281,6 +281,7 @@ services:
       - '@doctrine.orm.entity_manager'
       - 'beta'
       - '@=service("prestashop.adapter.multistore_feature").isActive()'
+      - '@prestashop.bundle.routing.converter.cache_provider'
```

Comment:
Hi @FabienPapet 

I tested the PR and I get this exception when I go to the feature flag page:

![Capture d’écran 2022-11-03 à 14 38 48](https://user-images.githubusercontent.com/13801017/199735827-cda3a0c3-d74c-45d7-a00c-ad3f31008a61.png)

which makes sens since there is also the `prestashop.admin.configure.advanced_parameters.stable_feature_flags_form_data_provider` which uses the same class `FeatureFlagsFormDataProvider` so it also needs `@prestashop.bundle.routing.converter.cache_provider`

Not a big deal to fix, but what worries me more is that it seems like you didn't test your PR?